src/main/java/examples/JsonSchemaExamples.java

```diff
@@ -28,4 +29,14 @@ public void validate(JsonSchema schema, Object json) {
       // Successful validation
     }
   }
+
+  public void instantiateWithCustomJsonFormatValidator() {
+    JsonSchemaOptions jsonSchemaOptionsWithFormatValidator = new JsonSchemaOptions()
+      .setBaseUri("https://vertx.io")
+      //Specify your own format validator here!
+      .setJsonFormatValidator(new DefaultJsonFormatValidatorImpl());
```

Comment:
Maybe we should implement a dummy example here, could be implemented via a lambda.